### DISCUSSION TITLE
Connecting a private blockchain to metamask

### DISCUSSION THREAD

**Discussion by LYAMZAYA** [ConsenSys/quorum#1478] [Q&A] [answered]
How can I connect Private Blockchain to MetaMask?Where do I get a new RPC URL, I didn't find a specific answer in the documentation.

**Comment by antonydenyer** [comment, marked as answer]
It depends how you're starting your node/network if you're using the defaults or the [quorum-dev-quickstart](https://github.com/ConsenSys/quorum-dev-quickstart) it will be `http://localhost:8545`

**Reply by LYAMZAYA** [reply]
I use bootnode, but MetaMask can't connet to my localhost

**Reply by antonydenyer** [reply]
what args are you using to start your node?

**Reply by LYAMZAYA** [reply]
First, in the terminal, I wrote `bootnode -boot boot.key,` then `bootnode -nodekey boot.key.`
Sorry, I'm just starting to work with this, I don't understand many things

**Reply by LYAMZAYA** [reply]
Then, `geth --networkid ""  --datadir "" --bootnodes enode:// --port 8545`

**Reply by antonydenyer** [reply]
Take a look at [quorum-dev-quickstart](https://github.com/ConsenSys/quorum-dev-quickstart) it's the best place for information about staring a private network.

**Reply by LYAMZAYA** [reply]
Is there something I can use on windows?

**Reply by LYAMZAYA** [reply]
Quorum only on linux, but i use windows, I've tried many ways to run bootnode, but metamask doesn't connect anyway

**Reply by antonydenyer** [reply]
I just wanted to confirm that what you are trying to do is have a private permissioned network running and not a personal node running that connects to a public ethereum network?

**Reply by LYAMZAYA** [reply]
Yes, I'm trying to launch a network isolated from Ethereum based on it

**Reply by antonydenyer** [reply]
In that case I think the easiest way to get started is to use [quorum-dev-quickstart](https://github.com/ConsenSys/quorum-dev-quickstart) on a linux machine. You should be able to start the docker containers without running the shell scripts so it should work on windows.  Another alternative is to use a hosted service like [QBS](https://consensys.net/quorum/qbs/)

**Reply by LYAMZAYA** [reply]
Thanks, I'll try what you suggested
